src/globus_sdk/_testing/models.py

```diff
@@ -161,6 +161,9 @@ def __iter__(
     ) -> t.Iterator[RegisteredResponse | ResponseList]:
         return iter(self._data.values())
 
+    def cases(self) -> t.Iterator[str]:
+        return iter(self._data.keys())
```

Comment:
The call to `.keys()` is not required. Iterating over the dict itself returns the same iterator (a `dict_keyiterator` object).

```suggestion
        return iter(self._data)
```

Reply:
And so it shall be! 😄